### PR TITLE
follow-unfollow functionality and some minor changes

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Hello, i have Added follow , unfollow user functionality, and have fixed some bugs, in social media app. kindly review it and let me know what do think about it !
Preview: https://blithe-in.netlify.app/
PR: https://github.com/kgajriaAgarwal/blithe-in/pull/9
files to check:
src/pages/UserProfile/ProfileDetails.js
src/pages/UserProfile/UserProfile.css
src/pages/UserProfile/UserProfile.js

### CODE REVIEW

**Review by b0llu — APPROVED** [review]
Great work on redux implementation. You can merge this PR

**Review by govindappaarun — APPROVED** [review]
Looks good to merge, 
remove commented code

**Comment by varun-Tad** [comment]
Code looks great. You can go ahead and merge

File: src/pages/Shared/NavBar/NavBar.js [hunk 1]
```diff
@@ -95,68 +94,60 @@ export const NavBar = (props) =>  {
...
+                        {/* underdevelopoment */}
+                        {/* <CustomTab icon={<Badge badgeContent={4} color="error">
+                            <Tooltip title="Feeds">
                                 <PublicIcon />
                             </Tooltip>
                         </Badge>} 
                         // label="Home" 
                         value="/"
-                        />
+                        /> */}
```
Comment (ScoobyDubeyDo) [inline src/pages/Shared/NavBar/NavBar.js:125]: **suggestion**: delete this section if not needed

**Review by ScoobyDubeyDo — COMMENTED** [review]
- feature working as expected
- Easy to understand
- Clean File structure

**Pull request merged**